Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css

```diff
@@ -24,7 +24,8 @@ span.bocAutoCompleteReferenceValue input
 /* Sets width for Chromium browsers */

```

Comment:
.blink already says that this is for chromium. If we want a comment, we should document what the workaround fixes.